astropy/table/tests/test_table.py

```diff
@@ -2158,7 +2159,15 @@ def test_masking(self, use_nullable_int):
                 else:
                     assert t2[name].dtype.kind == "i"
 
-                assert_array_equal(column.data, t2[name].data.astype(column.dtype))
+                with warnings.catch_warnings():
```

Comment:
This can be 
```
with np.errstate(invalid='ignore'):
    assert_...
```
(I think that's OK for all numpy versions.)

Reply:
Good idea. Applied. Thanks!